griptape/events/base_task_event.py

```diff
@@ -12,7 +13,7 @@ class BaseTaskEvent(BaseEvent, ABC):
     task_parent_ids: list[str] = field(kw_only=True, metadata={"serializable": True})
     task_child_ids: list[str] = field(kw_only=True, metadata={"serializable": True})
 
-    task_input: Union[BaseArtifact, BaseArtifact, tuple[BaseArtifact, ...]] = field(
-        kw_only=True, metadata={"serializable": True}
-    )
+    task_input: Union[
+        BaseArtifact, BaseArtifact, tuple[BaseArtifact, ...], tuple[BaseArtifact, Sequence[BaseArtifact]]
```

Comment:
Is that a duplicate `BaseArtifact`? I know not related to your change, but can we clean that up?